src/OrchardCore.Modules/OrchardCore.Lists/GraphQL/ContainedQueryObjectType.cs

```diff
@@ -8,10 +12,25 @@ public class ContainedQueryObjectType : ObjectGraphType<ContainedPart>
 {
     public ContainedQueryObjectType(IStringLocalizer<ContainedQueryObjectType> S)
     {
-        Name = "ContainedPart";
-        Description = S["Represents a link to the parent content item, and the order that content item is represented."];
+        Name = nameof(ContainedPart);
+        Description = S["Represents a link to the parent content item and the order in which the current content item is represented."];
 
         Field(x => x.ListContentItemId);
-        Field(x => x.Order);
+
+        Field<ContentItemInterface, ContentItem>("listContentItem")
+           .Description(S["the parent list content item"])
+           .ResolveLockedAsync(async x =>
+           {
+               var contentItemId = x.Source.ListContentItemId;
+               var contentManager = x.RequestServices.GetService<IContentManager>();
+
+               return await contentManager.GetAsync(contentItemId);
```

Comment:
```suggestion
           .ResolveLockedAsync(x =>
           {
               var contentItemId = x.Source.ListContentItemId;
               var contentManager = x.RequestServices.GetService<IContentManager>();

               return contentManager.GetAsync(contentItemId);
```

Reply:
`async/await` is required, because `ResolveLockedAsync` expects a `ValueTask`.

Reply:
oh. thanks